apolloschurchapp/scripts/add-packages.sh

```diff
@@ -26,5 +26,8 @@ else
 		PKGS=$(echo "$JSON" | sed -n "$DEPSLINE",/^$/p | grep "@apollosproject" | tr -d "\n")
 		DEVPKGS=$(echo "$JSON" | sed -n "$DEVDEPSLINE","$DEPSLINE"p | grep "@apollosproject" | tr -d "\n")
 fi
+
+PKGS=$(echo "$PKGS" | sed -e "s/@apollosproject\/react-native-airplay-btn@canary//")
```

Comment:
this would need to account for beta and latest as well, I'm fine just running from each dir and circling back around on this feature if you're cool with it.